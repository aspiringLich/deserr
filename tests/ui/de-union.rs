use deserr::Deserr;

#[derive(Deserr)]
#[deserr(error = deserr::Error)]
union Union {
    x: i32,
}

fn main() {}
