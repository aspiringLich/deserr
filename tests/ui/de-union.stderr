error: Unions aren't supported by the Deserr derive macro
 --> tests/ui/de-union.rs:5:1
  |
5 | union Union {
  | ^^^^^
